net-im/jabberd2: readd recently removed package

men2 opened the pull request:
Readd net-im/jabberd2 with the bugs fixed that caused the package to be
removed. Based on the previous 2.6.1 ebuild.

Closes: https://bugs.gentoo.org/656456
Bug: https://bugs.gentoo.org/629412
Bug: https://bugs.gentoo.org/631068
Bug: https://bugs.gentoo.org/614762
Package-Manager: Portage-2.3.24, Repoman-2.3.6

Zlogene commented on net-im/jabberd2/jabberd2-2.6.1-r1.ebuild:
```diff
@@ -0,0 +1,198 @@
+
+EAPI=6
+
+inherit autotools db-use eutils flag-o-matic pam systemd versionator
```
why `eutils`? why not `eapi7-ver` rather than versionator?

Zlogene commented on net-im/jabberd2/jabberd2-2.6.1-r1.ebuild:
```diff
@@ -0,0 +1,198 @@
+}
+
+src_prepare() {
+	epatch "${FILESDIR}/jabberd2-2.6.1-openssl-1.1.patch"
```
why not `eapply` ?

Zlogene commented on net-im/jabberd2/jabberd2-2.6.1-r1.ebuild:
```diff
@@ -0,0 +1,198 @@
+		tools/{migrate-jd14dir-2-sqlite.pl,pipe-auth.pl}
+
+	# remove useless upstart files wrt #498900
+	rm -rf "${ED%/}"/usr/etc
```
`|| die`

Zlogene commented on net-im/jabberd2/jabberd2-2.6.1-r1.ebuild:
```diff
@@ -0,0 +1,198 @@
+	# Fix systemd unit files installation path, bug #626026
+	emake systemddir="$(systemd_get_systemunitdir)" DESTDIR="${D}" install
+	einstalldocs
+	prune_libtool_files --modules
```
do not use this. `find "${D}" -name '*.la' -delete || die`

men2 commented:
Removed eutils. Seems it wasn't used.
Replaced versionator with eapi7-ver and use ver_test function instead.
Got rid of prune_libtool_files.
Added the missing || die.
